cmd/integration/commands/export_db_data.go

```diff
@@ -0,0 +1,584 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"math/big"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	libcommon "github.com/erigontech/erigon-lib/common"
+	"github.com/erigontech/erigon-lib/common/length"
+	"github.com/erigontech/erigon-lib/kv"
+	"github.com/erigontech/erigon/core/rawdb"
+	"github.com/erigontech/erigon/eth/consensuschain"
+	"github.com/erigontech/erigon/eth/ethconfig"
+	"github.com/erigontech/erigon/polygon/bor/borcfg"
+	"github.com/erigontech/erigon/polygon/bor/snaptype"
+	"github.com/erigontech/erigon/polygon/heimdall"
+	"github.com/erigontech/erigon/rlp"
+	"github.com/erigontech/erigon/turbo/debug"
+	"github.com/erigontech/erigon/turbo/snapshotsync/freezeblocks"
+)
+
+var cmdExportHeaderTd = &cobra.Command{
+	Use:   "export_header_td",
+	Short: "",
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := debug.SetupCobra(cmd, "integration")
+		label := kv.ChainDB
+		db, err := openDB(dbCfg(label, path.Join(datadirCli, label.String())), true, logger)
+		if err != nil {
+			logger.Error(err.Error())
```

Comment:
i would advise move all logic to a func like `doExportHeaderTD` - which will do usual `return err` and here log error once.